src/gentropy/dataset/summary_statistics.py

```diff
@@ -103,3 +104,25 @@ def exclude_region(self: SummaryStatistics, region: str) -> SummaryStatistics:
             ),
             _schema=SummaryStatistics.get_schema(),
         )
+
+    def get_locus_sumstats(
+        self: SummaryStatistics,
+        locus: DataFrame,
+        window: int,
+    ) -> DataFrame:
+        """Get summary statistics for a studyLocus.
+
+        Args:
+            locus (DataFrame): studyLocus DataFrame
+            window (int): Window size
+
+        Returns:
+            DataFrame: Summary statistics for the studyLocus region
+        """
+        chr = locus.select(f.first("chromosome")).collect()[0][0]
+        pos = locus.select(f.first("position")).collect()[0][0]
```

Comment:
I don't really like this, so, the input dataframe might have multiple rows, but only the coordinates of the first row is used, ritht? It is not really intuitive and I am afraid also a potential risk for surprises. This would not be an issue if the function would need chr, pos, study.